android/app/src/main/kotlin/com/jabook/app/jabook/audio/data/repository/SavedPlayerStateRepository.kt

```
@@ -42,6 +43,7 @@ public class SavedPlayerStateRepository
                 val entity = stateDao.getState(groupPath)
                 Result.Success(entity)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)